### PR TITLE
BAH-3339 |  Removed the log4j dependency and updated to slf4j

### PR DESCRIPTION
**Pull request by SanoferSameera**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -236,25 +229,16 @@
...
+                <artifactId>slf4j-api</artifactId>
+                <version>2.0.9</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-jcl</artifactId>
-                <version>${log4j.version}</version>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>1.3.0</version>
```
Comment (deepthi-mantena) [inline pom.xml:241]: The version we added here as dependency and the version we added above in line number 35 are different may be we can see if we can have same version for both to ensure that all components within the Logback library are compatible with each other.

Comment (SanoferSameera) [reply pom.xml:241]: Sure. I'm updating version in line 35

File: openerp-atomfeed-service/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </appender>
+
+    <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+        <target>SYSTEM_OUT</target>
+        <encoder>
+            <pattern>%d %-5p [openerp-atomfeed-service][%c] %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
```
Comment (gsluthra) [inline openerp-atomfeed-service/src/main/resources/logback.xml:24]: We might want to enable DEBUG level for service for some time.

File: package/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    </appender>
+
+    <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+        <target>SYSTEM_OUT</target>
+        <encoder>
+            <pattern>%d %-5p [openerp-atomfeed-service][%c] %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
```
Comment (gsluthra) [inline package/resources/logback.xml:24]: Debug level maybe until Odoo integration is under development?

File: openerp-atomfeed-service/src/main/resources/logback.xml (hunk 2 above)
Comment (SanoferSameera) [reply openerp-atomfeed-service/src/main/resources/logback.xml]: Done

File: package/resources/logback.xml (hunk 3 above)
Comment (SanoferSameera) [reply package/resources/logback.xml]: Done

**Review by deepthi-mantena — APPROVED** [review]

**Pull request merged**